sopel/coretasks.py

```diff
@@ -162,28 +221,41 @@ def execute_perform(bot, trigger):
 def startup(bot, trigger):
     """Do tasks related to connecting to the network.
 
-    001 RPL_WELCOME is from RFC2812 and is the first message that is sent after
-    the connection has been registered on the network.
+    ``001 RPL_WELCOME`` is from RFC2812 and is the first message that is sent
+    after the connection has been registered on the network.
 
-    251 RPL_LUSERCLIENT is a mandatory message that is sent after client
+    ``251 RPL_LUSERCLIENT`` is a mandatory message that is sent after client
     connects to the server in rfc1459. RFC2812 does not require it and all
     networks might not send it. We support both.
 
+    If ``sopel.irc.AbstractBot.connection_registered`` is set this function
+    does nothing and returns immediatly. Otherwise, the flag is set and the
+    function proceed normally to:
+
+    1. trigger auth method,
+    2. set bot's ``MODE`` (from ``core.modes``),
```

Comment:
```suggestion
    1. trigger auth method
    2. set bot's ``MODE`` (from ``core.modes``)
```

As above, dislike this style and the rest of the list didn't commit to it anyway.